docker-compose.yml

```diff
@@ -87,7 +87,6 @@ services:
       MEDIA_ROOT: ${MEDIA_ROOT:-}
     stdin_open: true
     tty: true
-    user: ${DOCKER_USER_ID}:${DOCKER_GROUP_ID}
```

Comment:
Among other services, having to set `user` on just this one was weird so I appreciate that it's gone.